src/content/reference/react-dom/server/renderToString.md

```diff
@@ -22,93 +22,93 @@ const html = renderToString(reactNode)
 
 ---
 
-## Reference {/*reference*/}
+## Başvuru dokümanı {/*reference*/}
 
 ### `renderToString(reactNode)` {/*rendertostring*/}
 
-On the server, call `renderToString` to render your app to HTML.
+Sunucuda, uygulamanızı HTML'ye dönüştürmek için `renderToString` çağrısı yapınız.
```

Comment:
Yabanci isim ve kisaltmalarda ekler yazilisa gore degil okunusa gore belirleniyor olmasi lazim. Mesela

`Shakespeare’in` seklinde yazariz. `Shakespeare’nin` degil.
Ya da `CNN'nin haberi` yazmak yerine `CNN'in haberi` seklinde yazariz.

O yuzden `HTML'e` seklinde guncellemek daha dogru olacaktir. Baska yerlerde de varsa hepsini guncellerseniz iyi olur.